src/webots/gui/WbVideoRecorder.cpp

```diff
@@ -177,10 +177,10 @@ void WbVideoRecorder::estimateMovieInfo(double basicTimeStep) {
 
   if (frameStep0 == 0 || abs(frameStep0 - EXPECTED_FRAME_STEP) > abs(frameStep1 - EXPECTED_FRAME_STEP)) {
     mMovieFPS = 1000.0 / frameStep1;
-    cDisplayRefresh = ceilRefresh * mVideoAcceleration;
```

Comment:
is the video acceleration used somewhere else than for displaying the label?
is this really correct?

Reply:
Good point, I tested and it was working, but now I am not so sure, I will retry.

Reply:
In any case, in the current state the `mVideoAcceleration` variable is useless and could be removed.